packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -46,6 +46,7 @@ export const messages = {
     'Enter desired directory (Press Enter to confirm or Esc to cancel)',
   parameter_gatherer_enter_username_name: 'Enter target username',
   parameter_gatherer_enter_alias_name: 'Enter a scratch org alias',
+  parameter_gatherer_enter_project_name: 'Enter project name',
```

Comment:
@ruthemmanuelle these messages were also here when I last pinged you but forgot to @mention you here. Just wanted to double check that you were able to take a look at this file as well.

Reply:
Thanks, @Sweetman . I hadn't seen these other messages, but they look fine.